src/ReactNativeZoomableView.tsx

```diff
@@ -1146,6 +1146,21 @@ class ReactNativeZoomableView extends Component<
     return true;
   }
 
+  /**
+   * Sets zoom relative to a "zoom subject" which is the relative coordinates
+   * of the view
+   */
+  zoomToZoomSubject(x: number, y: number, newZoomLevel: number) {
```

Comment:
also, fyi `zoomSubject` is used to refer to the whole transformed view used by this component rather than a particular coordinate.